src/Service/MagicLinks/ApplicationRiskBackgroundService.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Passwordless.Service.MagicLinks;
+
+public class ApplicationRiskBackgroundService : BackgroundService
```

Comment:
We will most likely not need a background service to implement quotas, so this can probably be removed